### PR TITLE
Fix login rendering in navbar

### PR DESCRIPTION
**Pull request by tinku-bansal**

### CODE REVIEW

**Review by KorolyovaLara — APPROVED** [review]
Seems all good.
I worked on dashboard too, but i will fix the code later to my version as it passes the username to access profile page
Screen Shot 2021-06-10 at 12 53 34 am

File: routes/api/gameApi.js [hunk 1]
```diff
@@ -13,7 +13,6 @@ const sequelize = require("../../config/connection");
  */
 
 router.get("/", withAuth, async (req, res) => {
-  console.log(req.user);
   try {
     const userId = req.user.id;
```
Comment (strahinjaajvaz) [inline routes/api/gameApi.js:18]: Can you please restructure the code?

`const { userId } = req.user;`

File: routes/api/gameApi.js [hunk 2]
```diff
@@ -29,11 +28,11 @@ router.get("/", withAuth, async (req, res) => {
       res.json(games);
     } catch (e) {
       console.log(e);
-      return res.status(409).json({ message: `The title already exists.` });
+      return res.status(409).json({ message: `Unexpected Error!` });
```
Comment (strahinjaajvaz) [inline routes/api/gameApi.js:31]: If this is an error, 409 is the wrong status code. 409 means unable to update.

File: routes/html/index.js [hunk 3]
```diff
@@ -5,35 +5,38 @@ const sequelize = require("../../config/connection");
 const router = require("express").Router();
 
 router.get("/", (req, res) => {
-  if (req.user) {
+  const loggedIn = req.user;
```
Comment (strahinjaajvaz) [inline routes/html/index.js:8]: `!!req.user`

File: routes/html/index.js [hunk 4]
```diff
@@ -5,35 +5,38 @@ const sequelize = require("../../config/connection");
...
+    res.render("dashboard", { loggedIn });
   } else {
     // render the landing page
     res.render("landingpage");
   }
 });
 
 router.get("/login", (req, res) => {
-  if (req.user) {
+  const loggedIn = req.user;
```
Comment (strahinjaajvaz) [inline routes/html/index.js:19]: Same as above

File: routes/html/index.js [hunk 5]
```diff
@@ -50,27 +53,28 @@ router.get("/user/:username", async (req, res) => {
...
     userInfo,
-    userLoggedIn,
     loggedIn,
   });
 });
 
 // get all games page
 router.get("/games", async (req, res) => {
   try {
+    const loggedIn = req.user;
```
Comment (strahinjaajvaz) [inline routes/html/index.js:71]: Same as above

File: routes/html/index.js [hunk 6]
```diff
@@ -5,35 +5,38 @@ const sequelize = require("../../config/connection");
...
-    res.render("dashboard", {});
+    res.render("dashboard", { loggedIn });
   } else {
     res.render("register", {});
   }
 });
 
 router.get("/user/:username", async (req, res) => {
-  const userLoggedIn = req.user;
+  const loggedIn = req.user;
```
Comment (strahinjaajvaz) [inline routes/html/index.js:39]: Same as above

File: routes/html/index.js [hunk 7]
```diff
@@ -5,35 +5,38 @@ const sequelize = require("../../config/connection");
...
-    res.render("dashboard", {});
+    res.render("dashboard", { loggedIn });
   } else {
     res.render("login", {});
   }
 });
 
 router.get("/register", (req, res) => {
-  if (req.user) {
+  const loggedIn = req.user;
```
Comment (strahinjaajvaz) [inline routes/html/index.js:29]: Same as above

File: routes/api/gameApi.js (hunk 1 above)
Comment (tinku-bansal) [reply routes/api/gameApi.js:18]: but user don't have UserID, user have 'id'

Comment (strahinjaajvaz) [reply routes/api/gameApi.js:18]: My bad! Awesome work! Sorry!

Comment (tinku-bansal) [reply routes/api/gameApi.js:18]: That's alright

File: routes/api/gameApi.js (hunk 2 above)
Comment (KorolyovaLara) [reply routes/api/gameApi.js:31]: will be fixed to 

      return res.status(500).json({ message: "Unexpected Error!" });

File: routes/html/index.js (hunk 3 above)
Comment (KorolyovaLara) [reply routes/html/index.js:8]: router.get("/", withAuth, async (req, res) => {
  const loggedIn = req.user;
  if (!!loggedIn) {
    // render the dashboard
    const fetchName = req.user.firstName;
    const fetchUsername = req.user.username;
    res.render("dashboard", { fetchName, fetchUsername, loggedIn });
  } else {
    // render the landing page
    res.render("landingpage");
  }
});

Comment (KorolyovaLara) [reply routes/html/index.js:8]: will be changed

```
router.get("/", withAuth, async (req, res) => {
  const loggedIn = req.user;
  if (!!loggedIn) {
```

**Pull request merged**